test/uitest.c

```diff
@@ -59,9 +59,11 @@ static int test_old()
     char pass[16];
     int ok = 0;
 
-    if ((ui_method =
-         UI_UTIL_wrap_read_pem_callback(test_pem_password_cb, 0)) == NULL
-        || (ui = UI_new_method(ui_method)) == NULL)
+    ui_method = UI_UTIL_wrap_read_pem_callback(test_pem_password_cb, 0);
+    if (!TEST_ptr(ui_method))
+        goto err;
+    ui = UI_new_method(ui_method);
+    if (!TEST_ptr(ui))
```

Comment:
I think it reasonable for the TEST_xxx macros to guarantee that each argument is used exactly once.  The failure messages will then provide more context.

I'll update the comment in testutil.h to reflect this.